src/newcharacter.cpp

```diff
@@ -187,6 +187,7 @@ void avatar::randomize( const bool random_scenario, points_left &points, bool pl
     init_age = rng( 16, 55 );
     // if adjusting min and max height from 145 and 200, make sure to see set_description()
     init_height = rng( 145, 200 );
+    blood_type = static_cast<blood_types>( rng( 0, 7 ) );
```

Comment:
You have two places selecting a random blood type.  Suggest factoring that out into a separate function.  Then it's easy to e.g. use the correct distribution of blood types, should we wish.